Nef_3/include/CGAL/Nef_3/SNC_constructor.h

```diff
@@ -24,12 +24,13 @@
 #include <CGAL/Nef_S2/Normalizing.h>
 #include <CGAL/Nef_3/bounded_side_3.h>
 #include <CGAL/Nef_3/Pluecker_line_3.h>
-#include <CGAL/Nef_3/SNC_decorator.h>
+#include <CGAL/Nef_3/SNC_const_decorator.h>
```

Comment:
So here we only need the const version?

Reply:
Yes since `SNC_decorator` includes `SNC_constructor`  (and SNC_constructor included SNC_decorator) I removed it to break the dual-cyclic include. Instead, the SNC_decorator is forward declared by SNC_structure which is included.

SNC_const_decorator can be included because it is self-contained.  I created an include graph for the whole Nef_3 package using: https://github.com/Leedehai/C-include-2-dot I can include it in the PR if needed.